DigitalLearningSolutions.Data/Models/CourseSettings.cs

```diff
@@ -38,22 +38,22 @@ public CourseSettings(string? settingsText)
                     ShowTime = (bool)settings["lm.st"];
                 }
 
-                if (settings.ContainsKey("lm.sl") && settings["lm.sl"] is bool)
+                if (settings.ContainsKey("lm.sl") && settings["lm.sl"] is bool showLearnStatus)
                 {
-                    ShowLearnStatus = (bool)settings["lm.sl"];
+                    ShowLearnStatus = showLearnStatus;
                 }
 
-                if (settings.ContainsKey("lm:ce") && settings["lm:ce"] is string)
+                if (settings.ContainsKey("lm.ce") && settings["lm.ce"] is string)

```

Comment:
It might be worth storing these strings as class constants, both to avoid having to type out constants multiple times and to avoid inconsistencies like this again. Maybe something like:
```
private const string BaseKey = "lm.";
private const string ConsolidationKey = $"{BaseKey}ce";
...
```
what do you think? 🙂

Reply:
Yeah i think that seems sensible